addons/base_import_module/models/ir_module.py

```diff
@@ -79,6 +79,8 @@ def _import_module(self, module, path, force=False, with_demo=False):
         if manifest_path:
             with file_open(manifest_path, 'rb', env=self.env) as f:
                 terp.update(ast.literal_eval(f.read().decode()))
+        if 'author' not in terp:
+            terp['author'] = "Odoo S.A."
```

Comment:
We don't want to apply this to all imported modules where the author is missing, only to industry modules